### DISCUSSION TITLE
MU-MIMO Simulation

### DISCUSSION THREAD

**Discussion by APoornima** [NVlabs/sionna#504] [Q&A]
Hello,

Currently I have SISO simulator with DFTsOFDM, I want to extend the same to simulate MU-MIMO, this is my first month of using Sionna, I have created a Channel scenario (Discrete Multipath) that is similar to TDL from Sionna, I am generating my own a, tau values, but this is for SISO.
How do I extend the same for MU-MIMO downlink scenario, with One BS with multiple antennas and multiple UEs with single antenna, or Multiple UEs with multiple antennas.

Thank you.

**Comment by SebastianCa** [comment]
Hi @APoornima,
it is difficult to give advice without knowing the exact setup. However, you need to define an antenna array according to your requirements. Please see [Antenna Arrays](https://nvlabs.github.io/sionna/examples/Sionna_tutorial_part3.html#Antenna-Arrays) and the [MIMO OFDM Tutorial](https://nvlabs.github.io/sionna/examples/MIMO_OFDM_Transmissions_over_CDL.html) for further details.
